Gemfile

```diff
@@ -26,12 +26,16 @@ group(:omnibus_package, :development, :test) do
   gem "pry"
   gem "rdoc"
   gem "yard"
-  gem "dep_selector"
   gem "guard"
   gem "cookstyle", ">= 2.0.0"
   gem "foodcritic", ">= 11.2"
 end
 
+group(:dep_selector) do
+  gem "dep_selector"
+  gem "dep-selector-libgecode"
+end
```

Comment:
gecode didn't build in hab and I didn't want to go down the rathole of figuring out what part of compiling C++ I needed to fix. Moved to a new group so it can be easily excluded.

Reply:
In what limited testing I've done, I've utilized the external gecode option and specify the path to hab's gecode.